Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/SyncKronosToShiftsController.cs

```diff
@@ -89,131 +88,146 @@ public ActionResult SyncKronosToShifts(string isRequestFromLogicApp)
 
             this.telemetryClient.TrackTrace($"SyncKronosToShifts starts at: {DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture)}", telemetryProps);
 
-            this.taskWrapper.Enqueue(this.ProcessKronosToShiftsShiftsAsync(isRequestFromLogicApp));
+            if (bool.TryParse(isRequestFromLogicApp, out bool isFromLogicApp))
+            {
+                this.taskWrapper.Enqueue(this.ProcessKronosToShiftsShiftsAsync(isRequestFromLogicApp));
 
-            using (StringContent stringContent = new StringContent(string.Empty))
+                using (StringContent stringContent = new StringContent(string.Empty))
+                {
+                    this.telemetryClient.TrackTrace($"SyncKronosToShifts ends at: {DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture)}", telemetryProps);
+                    return this.Ok(stringContent);
+                }
+            }
+            else
             {
-                this.telemetryClient.TrackTrace($"SyncKronosToShifts ends at: {DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture)}", telemetryProps);
-                return this.Ok(stringContent);
+                this.telemetryClient.TrackTrace($"Unable to call SyncKronosToShifts service at: {DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture)}. {Resource.IncorrectArgumentType}", telemetryProps);
+                return this.BadRequest(Resource.IncorrectArgumentType);
             }
         }
 
         /// <summary>
-        /// Process all the entities from Kronos to Shifts.
+        /// Process all the entities from kronos to shifts.
         /// </summary>
         /// <param name="isRequestFromLogicApp">true if the call is coming from logic app, false otherwise.</param>
         /// <returns>Returns task.</returns>
-#pragma warning disable CA1031 // Do not catch general exception types - Suppressing as there may be unknown points of failure.
         private async Task ProcessKronosToShiftsShiftsAsync(string isRequestFromLogicApp)
         {
-            var isOpenShiftRequestSyncSuccessful = false;
-            var isSwapShiftRequestSyncSuccessful = false;
-            var isMapPayCodeTimeOffReasonsSuccessful = false;
-
-            try
+             var isOpenShiftRequestSyncSuccessful = false;
+             var isSwapShiftRequestSyncSuccessful = false;
+             var isMapPayCodeTimeOffReasonsSuccessful = false;
+             try
             {
                 this.telemetryClient.TrackTrace($"{Resource.ProcessKronosToShiftsShiftsAsync} start at: {DateTime.UtcNow.ToString("O", CultureInfo.InvariantCulture)}" + " for isRequestFromLogicApp: " + isRequestFromLogicApp);
 
-                // Sync open shifts from Kronos to Shifts.
+                // Sync open shifts from kronos to shifts.
```

Comment:
The capitalization was correct. Fix it.
Consistently case as Kronos and Shifts.